lib/grape/validations/validators/coerce.rb

```diff
@@ -51,6 +58,14 @@ def valid_type?(val)
       end
 
       def coerce_value(val)
+        # JSON is not a type as Virtus understands it,
```

Comment:
As a future improvement maybe we should write a custom Virtus type for this, to be consistent and avoid special handling?